documentation/asciidoc/accessories/ai-camera/model-conversion.adoc

```diff
@@ -77,6 +77,8 @@ $ imxconv-pt -i <compressed ONNX model> -o <output folder>
 
 Both commands create an output folder that contains a memory usage report and a `packerOut.zip` file.
 
+For optimal use of the memory available to the accelerator on the IMX500 sensor, add `--no-input-persistency` to the above commands. However, this will disable the input tensor being generated and returned to the application for debugging purposes.
```

Comment:
Does "this will disable the input tensor being generated and returned to the application for debugging purposes." mean the same thing as "this will prevent the input tensor being returned to the application for debugging purposes." ?

Reply:
Not exactly, the input tensor can be generated, but not returned to the application.  The flag disables the input tensor generation entirely.

Reply:
```suggestion
For optimal use of the memory available to the accelerator on the IMX500 sensor, add `--no-input-persistency` to the above commands. However, this will disable input tensor generation and return to the application for debugging purposes.
```
Suggestion: minor wording improvement.